src/fixup.js

```diff
@@ -192,5 +193,11 @@
     document.body.appendChild(node);
   };
 
-  request.send();
+  request.onerror = function() {
+    console.error("Request to https://www.w3.org/TR/tr-outdated-spec failed.");
+  };
+
+  if (document.location.hostname === "www.w3.org") {
```

Comment:
I wonder if this `if` should wrap all of the above... you are setting up listeners, etc, that won't ever get fired.

Reply:
I thought it wouldn't have any impact on performance but ok to move *all* the code inside the `if`